### PR TITLE
[3.10] Installation languages for ru-RU, it-IT and sk-SK

### PR DESCRIPTION
**Pull request by Bakual**
@zero-24 Merge when you feel ready 😄

### CODE REVIEW

**Comment by zero-24** [comment]
Thanks will merge here after the release.

**Comment by infograf768** [comment]
I guess this can be merged now as release is postponed...

**Comment by infograf768** [comment]
WE have another one waiting https://forum.joomla.org/viewtopic.php?f=54&t=990261&start=30#p3646948

**Comment by zero-24** [comment]
Yes please add it to this PR and I'm happy to merge it into the release.

File: installation/language/sk-SK/sk-SK.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<metafile version="3.10" client="installation">
+<metafile version="3.9" client="installation">
```
Comment (zero-24) [inline installation/language/sk-SK/sk-SK.xml]:
```suggestion
<metafile version="3.10" client="installation">
```

**Comment by zero-24** [comment]
Merged now. Thanks!

**Pull request merged**